### ISSUE TITLE
source for call number and location is sometimes incorrect

### ISSUE THREAD

**Issue by joycebcat** [pulibrary/orangelight#124]
In testing the site today (May 5th), I found a record "Asians in Connecticut" which is pulling both location and call number from the 852 in the bib record.  This information does not apply to Princeton.  The location and call number for our data MUST ALWAYS come from the holdings record.

**Comment by tampakis** [comment]
Record `3478680` doesn't have a holdings record. I looked at the bibdata code and saw that the bib 852 only gets stripped if it has an associated holdings record. I changed the code so now the bib 852 gets stripped regardless of there being a holdings record. It should be reflected in the next full voyager dump I do (probably in a few weeks).

**Comment by joycebcat** [comment]
Ok, sounds good





On May 5, 2015, at 1:00 PM, tampakis <notifications@github.com<mailto:notifications@github.com>> wrote:


Record 3478680 doesn't have a holdings record. I looked at the bibdata code and saw that the bib 852 only gets stripped if it has an associated holdings record. I changed the code so now the bib 852 gets stripped regardless of there being a holdings record. It should be reflected in the next full voyager dump I do (probably in a few weeks).
